src/bertron_schema/schema/bertron_schema.yaml

```diff
@@ -1,96 +1,288 @@
----
 id: https://w3id.org/ber-data/bertron-schema
 name: bertron-schema
-title: bertron-schema
-description: |-
-  LinkML schema for ber data intergration work
+title: BERtron schema
+description: Schema for BERtron common data model.
+version: 0.0.1
 license: BSD-3
 see_also:
   - https://ber-data.github.io/bertron-schema
-
 prefixes:
-  bertron_schema: https://w3id.org/ber-data/bertron-schema/
+  bertron: https://w3id.org/ber-data/bertron-schema/
   linkml: https://w3id.org/linkml/
-  biolink: https://w3id.org/biolink/
   schema: http://schema.org/
-  PATO: http://purl.obolibrary.org/obo/PATO_
-  example: https://example.org/
-default_prefix: bertron_schema
+default_prefix: bertron
 default_range: string
-
+default_curi_maps:
+  - semweb_context
 imports:
-  - linkml:types
+  - bertron_types
 
 classes:
 
-  NamedThing:
-    description: >-
-      A generic grouping for any identifiable entity
-    slots:
-      - id
-      - name
-      - description
+  Entity:
     class_uri: schema:Thing
+    description: |
+      An object retrieved by BERtron from a BER data API.
+    attributes:
+      ber_data_source:
+        description: The BER member from whence the entity originated.
+        range: BERSourceType
+        required: true
+      # TODO!
+      credit_metadata:
+        description: Citation metadata for the entity.
+        todos:
+        - add in KBase credit metadata schema
```

Comment:
basic elements of citation metadata:

- permanent resolvable ID(s)
- title / name of dataset
- contributor list
- repository where data is stored/published/etc.
- version info / publication/access date